src/screens/LoginPage/LoginPage.tsx

```diff
@@ -189,6 +189,8 @@ function LoginPage(): JSX.Element {
         }
       } else {
         toast.warn('User not found!');
+        // toast.warn('User not found! kindly relogin as sessison has expired');
+        // window.location.replace('/login');
```

Comment:
Please remove the commented code if no longer being used.

Reply:
Will do. Thanks